.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,39 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve 🤔.
-labels: bug
+labels: [bug]
+title: ''
+assignees: ''
 ---
 
 <!-- ⚠️ If you do not respect this template, your issue will be closed -->
 <!-- ⚠️ Make sure to browse the opened and closed issues -->
 
-### Steps to reproduce the problem
+### System information
+<!-- Please, provide some information about your system for debugging purposes. -->
+- Device: (e.g. MacBook Pro (16-inch, 2019))
+- OS: (e.g. macOS Monterey version 12.3.1)
+- Python version: (e.g. 3.8.9)
+- Prototype version: (e.g. 1.0.0)
```

Comment:
It'd be nice if these examples were included in the HTML comment but not in the actual text.  I could imagine people just submitting without modifying the text.

Also, can you think of a circumstance in which the device is relevant?  It might be if they are running on POWER or Z or something, but for the most part I wouldn't expect it to be.

Reply:
> It'd be nice if these examples were included in the HTML comment but not in the actual text. I could imagine people just submitting without modifying the text.

Agreed! I'll take care of it.


> Also, can you think of a circumstance in which the device is relevant? It might be if they are running on POWER or Z or something, but for the most part I wouldn't expect it to be.

Yeah, I see what you mean, however I don't think there is much harm in asking, just in case. It's probably not a huge effort for the user either. What do you think?